fhttp/httprunner_test.go

```diff
@@ -89,3 +94,29 @@ func TestHTTPRunnerBadServer(t *testing.T) {
 	}
 	log.Infof("Got expected error from mismatch/bad server: %v", err)
 }
+
+// need to be the last test as it installs Serve() which would make
+// the error test for / url above fail:
+
+func TestServe(t *testing.T) {
```

Comment:
instead of doing all of the logic ourselves, could we use `httptest.NewServer` instead (see: http://devdocs.io/go/net/http/httptest/index#NewServer)?

Reply:
the test is to test serve and it just gets a local free port first
(but I added todo on how fhttp.Serve can be improved)